feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -395,40 +395,36 @@ func TestComponentParent(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
-			t.Logf("Found card list for Component Type %v : %v", tc.componentType, cardList[tc.desc])
+			t.Logf("Found Component list for Type %v : %v", tc.componentType, cardList[tc.desc])
 			if len(cardList[tc.desc]) == 0 {
-				t.Fatalf("Get Card list for %q: got 0, want > 0", dut.Model())
+				t.Fatalf("Get Component list for %q: got 0, want > 0", dut.Model())
 			}
 			// Validate parent component
 			for _, card := range cardList[tc.desc] {
-				t.Logf("Validate card %s", card)
+				t.Logf("Validate component %s", card)
 				cardName := card
```

Comment:
Local variable 'cardName' is getting set to 'parentName' later within the 'for' loop. Wont be able to use the loop index variable 'card' in place of the local variable 'cardName'. Will rename to card->comp though. Is that ok?
Line #422: cardName = parentName

Reply:
It doesn't matter that cardName is being set to parentName later on. You can still set the loop variable to a new value. It would be equivalent to what you have now. If you don't like the style of doing that, please rename "cardName" to "curr"